docs/userguide/performance.rst

```diff
@@ -1,7 +1,7 @@
 .. _label-performance:
 
-Performance and Scalability
-===========================
+Historical: Performance and Scalability
+=======================================
 
```

Comment:
Is there any value in giving a sentence or two of context of why it's historical?  When I'm new to a subject or project, I'm sometimes frustrated to see "historical" because the now-old way of doing things often matches my naive understanding of the data structures or expected workflow.

Reply:
I added a justification note here.